### PR TITLE
Return Error if invalid from and to params (closes #1441)

### LINKED ISSUE #1441: Don't receive error GET /api/v2/trades with invalid from/to params
Send a get request to http://api.wb.local/api/v2/trades
Add valid market and 'from' and 'to' invalid params (e.g. from=0&to=0 or from=10&to=-20)

Now: user get a response back, an empty array. Note to: if the from param has a greater than zero param, it will retrieve data.

Expected: user should not get a 200 response back and a meaningful error message should appear

### PR DESCRIPTION
**Pull request by kaiten-mahajan**
Return Error if invalid from and to params #1441

### CODE REVIEW

File: app/api/api_v2/trades.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Trades < Grape::API
       use :market, :trade_filters
     end
     get "/trades" do
+      validate_trade_params(params[:from], params[:to]) if params[:from].present? && params[:to].present?
```
Comment (ysv) [inline app/api/api_v2/trades.rb]: All parameters validations should be in `params`

File: app/api/api_v2/helpers.rb [hunk 2]
```diff
@@ -119,5 +119,9 @@ def get_k_json
         JSON.parse('[%s]' % redis.lrange(key, offset, -1).join(','))
       end
     end
+
+    def validate_trade_params(from, to)
+      raise Error.new(text: "from should be less than to", status: 422) if from > to
```
Comment (ysv) [inline app/api/api_v2/helpers.rb]: Use single quotes
`'From should be less than to'`

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by ysv** [comment]
@kaiten-mahajan Please update

**Comment by kaiten-mahajan** [comment]
@ysv I'm still working on this.

File: app/api/api_v2/trades.rb (hunk 1 above)
Comment (kaiten-mahajan) [reply app/api/api_v2/trades.rb]: @ysv done

**Comment by kaiten-mahajan** [comment]
@ysv I have made the changes. Please have a look now.

File: spec/api/api_v2/trades_spec.rb [hunk 3]
```diff
@@ -95,6 +95,14 @@
...
     end
+
+    it 'should validate from and to param' do
+      another = create(:trade, bid: bid)
+
+      get '/api/v2/trades', market: 'btcusd', from: another.id, to: ask_trade.id
+      expect(response.code).to eq '422'
+      expect(response.body).to eq '{"error":{"code":1001,"message":"from should be less than to."}}'
+    end
   end
```
Comment (ysv) [inline spec/api/api_v2/trades_spec.rb]: @kaiten-mahajan can you add spec with only from and only to parameter

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by kaiten-mahajan** [comment]
@ysv Done.

File: spec/api/api_v2/trades_spec.rb (hunk 3 above)
Comment (kaiten-mahajan) [reply spec/api/api_v2/trades_spec.rb]: @ysv Done.

**Pull request merged**